core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -1258,50 +1258,46 @@ protected function getNewPosition($mode, $pid=null, $insertInto=false)
 						$newPID = $objSorting->pid;
 						$curSorting = $objSorting->sorting;
 
-						// Do not proceed without a parent ID
-						if (is_numeric($newPID))
```

Comment:
As @ausi pointed out, wouldn't it be much easier to use `is_numeric($newPID) || $newPID === null` here? This would solve the problem in a less invasive way.

Reply:
you mean instead of removing the if-condition? I don't think this was ever intended to catch a string-value

Reply:
Yes, keeping the if condition and allowing `null` in addition to numeric values.

Reply:
even though the comment explicitly said to check for "without a parent ID"??

Reply:
`0` and `null` are valid parent IDs, but the empty string is not I think, so the comment would be somewhat correct.
If the `if` is needed at all is hard to tell IMO, therefore I’d prever the less invasive way with the minimal change.